### ISSUE TITLE
New translation: {Croatian}

### ISSUE THREAD

**Issue by slavenabd** [squidfunk/mkdocs-material#875]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "hr",
  "clipboard.copy": "Kopiraj u međuspremnik",
  "clipboard.copied": "Kopirano u međuspremnik",
  "edit.link.title": "Uredi stranicu",
  "footer.previous": "Prethodno",
  "footer.next": "Sljedeće",
  "meta.comments": "Komentari",
  "meta.source": "Izvor",
  "search.language": "hr",
  "search.placeholder": "Pretraživanje",
  "search.result.placeholder": "Unesite pojam pretraživanja",
  "search.result.none": "Ništa nije pronađeno",
  "search.result.one": "1 rezultat pretraživanja",
  "search.result.other": "# rezultata pretraživanja",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Preskočite na sadržaj",
  "source.link.title": "Idi u repozitorij",
  "toc.title": "Sadržaj"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by slavenabd** [comment]
I would suggest updating the following sections:
```
  "clipboard.copy": "Kopirajte u međuspremnik",
  "source.link.title": "Idite u repozitorij",
```
It will match the formal tone used in other sections: 
```
  "search.result.placeholder": "Unesite pojam pretraživanja",
  "skip.link.title": "Preskočite na sadržaj",
```
Thanks

**Comment by squidfunk** [comment]
Added as part of 330229f.

**Comment by squidfunk** [comment]
Released as part of 3.0.5
